server/libs/infra/src/repositories/typesense.repository.ts

```diff
@@ -216,6 +216,12 @@ export class TypesenseRepository implements ISearchRepository {
     await this.delete(SearchCollection.FACES, ids);
   }
 
+  async deleteAllFaces(): Promise<void> {
+    const faceCollection = schemaMap[SearchCollection.FACES].name;
+    const records = await this.client.collections(faceCollection).documents().delete({ filter_by: 'ownerId:!=null' });
```

Comment:
Why do we need the filter here? Also, maybe it would be good to have a general delete collection method where search collection is passed in. I think being able to clear the asset collection may be useful.

Reply:
I tried to delete the collection and recreate it, but somehow the process of inserting a document into the collection happened before it was recreated and caused an error.